index.bs

```diff
@@ -1960,22 +1981,14 @@ Instances of <code>ReadableByteStream</code> are created with the internal slots
     <td>A value indicating how the stream failed, to be given as a failure reason or exception when trying to operate
       on an errored stream
   </tr>
-  <tr>
-    <td>\[[underlyingByteSource]]
```

Comment:
Oh, I see. You just moved it to the controller. Interesting! I wonder if we can/should do that for ReadableStream...